modules/apps/packet-forward-middleware/keeper/migrator_test.go

```diff
@@ -0,0 +1,146 @@
+package keeper_test
+
+import (
+	"encoding/json"
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	pfmkeeper "github.com/cosmos/ibc-go/v10/modules/apps/packet-forward-middleware/keeper"
+	v3 "github.com/cosmos/ibc-go/v10/modules/apps/packet-forward-middleware/migrations/v3"
+	pfmtypes "github.com/cosmos/ibc-go/v10/modules/apps/packet-forward-middleware/types"
+	transfertypes "github.com/cosmos/ibc-go/v10/modules/apps/transfer/types"
+	ibctesting "github.com/cosmos/ibc-go/v10/testing"
+)
+
+func (s *KeeperTestSuite) TestMigrator() {
+	retries := uint8(2)
+	var (
+		accA, accB, accC, port string
+		firstHopMetadata       *pfmtypes.PacketMetadata
+		err                    error
+		nxtBz                  []byte
```

Comment:
nit: doesn't feel like we save much of anything by removing the one letter here. would prefer "nextBz"